README.md

```diff
@@ -53,7 +55,7 @@ error handler:
 const http = require('http');
 const writeErrorToResponse = require('strong-error-handler')
   .writeErrorToResponse;
-const errHandlingOptions = {debug: process.env.NODE_ENV === 'development'}
+const errHandlingOptions = { debug: process.env.NODE_ENV === 'development' };
```

Comment:
Unrelated change, it's also changing from LoopBack style (no spaces around brackets) to Node.js style.

Please revert.